src/dsync/dsync.c

```diff
@@ -2840,8 +2840,9 @@ int main(int argc, char **argv)
     /* verbose by default */
     mfu_debug_level = MFU_LOG_VERBOSE;
 
-    /* By default, sync option will preserve all attributes. */
+    /* By default, sync option will preserve most attributes. */
     copy_opts->preserve = true;
+    copy_opts->preserve_xattrs = true;
```

Comment:
```suggestion
    copy_opts->preserve_xattrs = false;
```
Do you want this _**off**_ by default?

Reply:
Yes I do want if off by default, that's thw whole point of this PR.
Copying xattrs by default messes up to many things.

And since rsync -a (which is the most commonly used flag) does not copy xattrs (nor ACLs btw) so should dsync.
I think dsync should mimic rsync as much as possible since that is the "gold standard" at the moment.

My use case for this is migrating a Lustre file system without PFL to a new shiny system with PFL and I do not want  the layout to be copied over.

Frankly I'm a bit surprised to see that dsync did not try to mimic rsync in the first place when it comes to options and behavior.

Reply:
Yes, I understand the use case, but it looks like `copy_opts->preserve_xattrs = true;` turns this _**on**_ by default, not _**off**_.
From what I've gathered from conversations with Adam Moody, it generally seems desirable to mimic the serial versions of the tools, but considering simplicity and development effort, this just hasn't yet been done in all places.

Reply:
oops, now I see what you mean, that's a line that shouldn't be there....
I made a bobo when moving the patch onto master, my original is based in 0.10.1...
I'll drop that.

Reply:
fixed